.changeset/dull-knives-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
```

Comment:
regressions dont need changesets

Reply:
Got it. Will remove it. But looks like we have a documentation problem then: 
![Screenshot_2024-07-02-10-58-49-206_com.android.chrome-edit.jpg](https://github.com/RocketChat/Rocket.Chat/assets/43561537/5d978122-2cb0-44ae-b65a-c32156154998)